.github/workflows/lint.yaml

```diff
@@ -15,22 +15,22 @@ jobs:
       checks: write
     steps:
       - name: Checkout
-        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
 
-      - name: Toolchain
+      - name: Toolchain # TODO: remove this one
```

Comment:
related to discussion here: https://bravesoftware.slack.com/archives/C8ZJB9F9B/p1696264508777859